src/Array.ts

```diff
@@ -1105,7 +1105,7 @@ export function uniq<A>(E: Eq<A>): (as: Array<A>) => Array<A> {
  * @since 2.0.0
  */
 export function sortBy<A>(ords: Array<Ord<A>>): (as: Array<A>) => Array<A> {
```

Comment:
I think it would be better to require `NonEmptyArray<Ord<A>>` instead of noop on empty array.

Reply:
Well, I would rather incline towards handling this case by this function rather than forcing user to provide `NonEmptyArray`. Besides, won't it break compatibility?

Reply:
> won't it break compatibility?

Yes it will, @raveclassic this fix LGTM

Reply:
@gcanti ok then, lgtm too